contrib/datawave-quickstart/bin/services/hadoop/install.sh

```diff
@@ -89,13 +89,10 @@ else
 fi
 
 # Ensure that $JAVA_HOME is observed by all hadoop scripts
-sed -i "s|.*\(export JAVA_HOME=\).*|\1${JAVA_HOME}|g" ${HADOOP_CONF_DIR}/hadoop-env.sh
+sed -i'.bak' "s|.*\(export JAVA_HOME=\).*|\1${JAVA_HOME}|g" ${HADOOP_CONF_DIR}/hadoop-env.sh && rm ${HADOOP_CONF_DIR}/hadoop-env.sh.bak
 
 verifySSHConfig
 
-assertCreateDir "${DW_HADOOP_MR_INTER_DIR}"
-assertCreateDir "${DW_HADOOP_MR_DONE_DIR}"
-
```

Comment:
What's the reason for removing these lines?

Reply:
They're not needed because the directories are on HDFS - `assertCreateDir` checks local filesystem directories.

Reply:
Ah, sounds good. Thanks.